velox/exec/tests/NestedLoopJoinTest.cpp

```diff
@@ -81,15 +81,8 @@ class NestedLoopJoinTest : public HiveConnectorTestBase {
       const std::vector<RowVectorPtr>& probeVectors,
       const std::vector<RowVectorPtr>& buildVectors,
       int32_t numDrivers = 1) {
```

Comment:
Just curious : You need to increase the number of drivers to hit this issue. Where are you changing that number ?

Reply:
https://github.com/facebookincubator/velox/blob/fc9b91b351c5befe2fc31eebeeb59d773457ffc0/velox/exec/tests/NestedLoopJoinTest.cpp#L379

Reply:
Seems the multi-driver config should be tested for more cases, I updated `runTest` to test with both one driver and multi-driver config.